NDTensors/ext/NDTensorsCUDAExt/typeparameteraccessors.jl

```diff
@@ -0,0 +1,20 @@
+# # `TypeParameterAccessors.jl` overloads.
+using NDTensors.TypeParameterAccessors: TypeParameterAccessors, Position, parameter
+## TODO this seems like a `GPUArrays` generic function
+storagemode(t::Type{<:CuArray}) = parameter(t, 3)
```

Comment:
```suggestion
storagemode(t::Type{<:CuArray}) = parameter(t, Position(3))
```
to ensure type stability?

Reply:
Actually, it seems nice to define this as:
```julia
storagemode(t::Type{<:CuArray}) = parameter(t, storagemode)
```
since we are already defining the position of `storagemode`.

Reply:
I defined this using storagemode, thanks!